app/org/sagebionetworks/bridge/services/UploadService.java

```diff
@@ -220,6 +220,29 @@ public Upload getUpload(@Nonnull String uploadId) {
             @Nullable DateTime startTime, @Nullable DateTime endTime) {
         checkNotNull(healthCode);
         
+        return getUploads(startTime, endTime, true, (start, end)-> {
+            return uploadDao.getUploads(healthCode, start, end);
+        });
+    }
+    
+    /**
+     * <p>Get uploads for an entire study in a time window. Start and end time are optional. If neither are provided, they 
+     * default to the last day of uploads. If end time is not provided, the query ends at the time of the request. If the 
+     * start time is not provided, it defaults to a day before the end time. The time window is constrained to two days 
+     * of uploads (though those days can be any period in time). </p>
+     */
+    public DateTimeRangeResourceList<? extends UploadView> getStudyUploads(@Nonnull StudyIdentifier studyId,
+            @Nullable DateTime startTime, @Nullable DateTime endTime) {
+        checkNotNull(studyId);
+
+        return getUploads(startTime, endTime, false, (start, end)-> {
```

Comment:
I hadn't thought about it but creating a report of uploads by their schema does seem useful, I'll add.